Sources/SwiftTUI/RunLoop/Application.swift

```diff
@@ -11,6 +11,8 @@ public class Application {
     private var invalidatedNodes: [Node] = []
     private var updateScheduled = false
 
+    private var stdInHandle: FileHandle?
```

Comment:
```suggestion
```

Looks like this is not needed anymore

Reply:
Oh, nice catch. Let me remove that cruft.